src/groups/mqb/mqbmock/mqbmock_domain.cpp

```diff
@@ -149,6 +149,16 @@ int Domain::processCommand(mqbcmd::DomainResult*        result,
     return -1;
 }
 
+void Domain::removeDomainStart()
+{
+    BSLS_ASSERT_SAFE(false && "NOT IMPLEMENTED!");
```

Comment:
I think these may be better as `// NOTHING`.  This is a mock, so it probably shouldn't assert when two of its functions are called.